ROADMAP.md

```diff
@@ -1,18 +1,56 @@
-# Graphene Roadmap
+# GraphQL Python Roadmap
 
-In order to move Graphene and the GraphQL Python ecosystem forward I realized is essential to be clear with the community on next steps, so we can move uniformly.
+In order to move Graphene and the GraphQL Python ecosystem forward it's essential to be clear with the community on next steps, so we can move uniformly.
 
 _👋 If you have more ideas on how to move the Graphene ecosystem forward, don't hesistate to [open a PR](https://github.com/graphql-python/graphene/edit/master/ROADMAP.md)_
 
-## Graphene 3
 
-The goal is to summarize the different improvements that Graphene will need to accomplish for version 3.
+## Now
+- [ ] Continue to support v2.x with security releases
+- [ ] Last major/feature release is cut and graphene-* libraries should pin to that version number
 
-In a nushell, Graphene 3 should take the Python 3 integration one step forward while still maintaining compatibility with Python 2.
+## Next
+New features will only be developed on version 3 of ecosystem libraries.
 
-- 🚀 [graphql-core-next](https://github.com/graphql-python/graphql-core-next) GraphQL engine support (almost same API as graphql-core)
-- 🔸 GraphQL types from type annotations - [See issue](https://github.com/graphql-python/graphene/issues/729)
-- 📄 Schema creation from SDL (API TBD)
-- ✨ Improve connections structure
-- 📗 Improve function documentation
-- 🔀 Add support for coroutines in Connection, Mutation (abstracting out Promise requirement) - [See PR](https://github.com/graphql-python/graphene/pull/824)
+### [Core-Next](https://github.com/graphql-python/graphql-core-next)
+Targeted as v3 of [graphql-core](https://pypi.org/project/graphql-core/), Python 3 only
+
+### Graphene
+- [ ] Integrate with the core-next API and resolve all breaking changes
+- [ ] GraphQL types from type annotations - [See issue](https://github.com/graphql-python/graphene/issues/729)
+- [ ] Schema creation from SDL (API TBD)
```

Comment:
do we want this, we now have Ariadne and Tartiflette that are schema first libraries for GraphQL, might be worth focusing on other things :)

Reply:
Yeah I agree, I think it would be better to focus time on programmatic schema generation and direct people to other libraries if they want to build their server using the SDL.